vendor/github.com/aws/aws-sdk-go/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Why are dependencies being committed?

Reply:
This is for use with [godep](https://github.com/tools/godep) which should make cloning and developing the project easier.

Reply:
How hard was go get ./...? :p

Reply:
Harder than one `go get` and a `godep restore` :)

Reply:
Put this in a separate PR. This PR has too many unrelated changes.

Reply:
Neel I will fly to Atlanta and throw you into Florida.

Reply:
wait what the hell

Reply:
Yeah, I'm not approving this with godep in it. The Go package manager should do this. 🤞 2.0

Reply:
.___________.